test/cases/oscap.sh

```diff
@@ -77,12 +77,12 @@ SSH_KEY=${SSH_DATA_DIR}/id_rsa
 SSH_KEY_PUB=$(cat "${SSH_KEY}".pub)
 
 case "${ID}-${VERSION_ID}" in
-    "rhel-8.8")
+    "rhel-8.8" | "rhel-8.9")
```

Comment:
I think we should use wildcards here, like in this script: https://github.com/osbuild/osbuild-composer/blob/055a63d71f1d49ebc894f3d56f1e67be058d6d61/tools/libvirt_test.sh#L274

Something for a follow-up, though. :)